english/hands/door/door.md

```diff
@@ -1,5 +1,6 @@
 After having watched MacGyver in your childhood, you are able to build up a door with the rope by kindle the fire
 with the lighter.
 
-Finally, the marshmallows start to melt down and you see a beautiful beach with two moons in the sky. You have travel
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.